### PR TITLE
fix: update RegExp

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [x] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | fix: update RegExp |
| 🇨🇳 Chinese | fix: update RegExp |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by afc163** [comment]
ci broken, please check it.

File: .jest.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const transformIgnorePatterns = [
 
 function getTestRegex(libDir) {
   if (libDir === 'dist') {
-    return 'demo\\.test\\.js$';
+    return 'demo\\.test\\.(j|t)s$';
```
Comment (yoyo837) [inline .jest.js:10]: 会不会 `(`、`|`、`)`都需要双反斜杠?

Comment (afc163) [inline .jest.js:10]: 这不是正则的写法

```suggestion
    return 'demo\\.test\\.[jt]s$';
```

Comment (li-jia-nan) [reply .jest.js]: () 会分组，[] 没有额外创建分组，所以第一个是两个捕获内容，第二个就一个整体，这里还是应该用 () 吧

Comment (afc163) [reply .jest.js]: 哦对的，应该是 ()

**Comment by afc163** [comment]
trace #37863

**Pull request closed without merging**